operators/src/error.rs

```diff
@@ -193,6 +193,12 @@ pub enum Error {
 
     TemporalRasterAggregationLastValidRequiresNoData,
     TemporalRasterAggregationFirstValidRequiresNoData,
+
+    LoadingInfo {
+        reason: String, // TODO: find cleaner way to make errors in services crate visible
```

Comment:
maybe just a box error?

Reply:
the error occurs in the services crate so the operators Error cannot know the error and thus also a `Box<geoengine_services::Error>` won't work

Reply:
maybe use Trait std::error::Error or Into Trait std::error::Error

Reply:
`std::error::Error` is not Send + Sync und thus our `Result` would no longer be Send + Sync. Also our own `Error` doesn't even implement `std::error::Error` or does it?

Reply:
you can do:

```
#[snafu(display("LoadingInfoError: {}", source))]
    LoadingInfo {
        source: Box<dyn std::error::Error + Send + Sync>, // TODO: find cleaner way to make errors in services crate visible
    },

Reply:
4e19dc43043d20988a199028294ebe05e3ea7533